.circleci/config.yml

```diff
@@ -158,16 +176,13 @@ jobs:
       - run:
           name: Build CLI
           command: yarn build
-      - run:
-          name: Bundle CLI
-          command: yarn build:bundle
       - run:
           name: Compress CLI bundle
-          command: >
-            cp -av ./raiden bundle/ ;
-            tar -czvf raiden-cli.tar.gz bundle --transform='s/^bundle/raiden-cli/'
+          command: |
+            cp --verbose ./raiden ./package.json ./build/
+            tar --create --gzip --verbose --file raiden-cli.tgz ./build --transform='s/^build/raiden-cli/'
       - store_artifacts:
-          path: raiden-cli.tar.gz
+          path: raiden-cli.tgz
```

Comment:
This is still publishing from the `tgz`. Just to record what we've talked on chat, I think it'd be interesting to revert the artifact publishing to the `bundle`, but `npm` publishing to be done with `package.json`'s `files` array covering the `build` folder. `bin` field with `raiden-cli` pointing to `build/index.js` may be needed to have node put the bin in place where needed, although this may require the `shebang` in `index.js` and maybe that it's made executable post-build.

Reply:
For reference: https://blog.npmjs.org/post/118810260230/building-a-simple-command-line-tool-with-npm.html

Reply:
✔️